packages/adapters/src/process-instance-adapter.ts

```diff
@@ -149,6 +157,11 @@ IComponent {
 
         if (statusCode > 0) {
             this.logger.debug("Process returned non-zero status code", statusCode);
+
+            if (!development()) {
+                this.logger.debug("Forwarding stdout & stderr from runner:");
+                createReadStream(runnerLogFile).pipe(process.stderr);
```

Comment:
btw: do you need to later (after the CSI Controller exits) unpipe the createReadStream(runnerLogFile).pipe(process.stderr); ?

Reply:
Yes, it would seem so... I'll check this, as we've got some hanging pipes in multiple locations and I see warnings about listeners piling up in a couple places.